src/android/CameraLauncher.java

```diff
@@ -1261,14 +1261,23 @@ public void processPicture(Bitmap bitmap, int encodingType) {
         CompressFormat compressFormat = encodingType == JPEG ?
                 CompressFormat.JPEG :
                 CompressFormat.PNG;
+        String imageMimeType = encodingType == JPEG ?
+                JPEG_MIME_TYPE :
+                PNG_MIME_TYPE;
 
         try {
             if (bitmap.compress(compressFormat, mQuality, jpeg_data)) {
                 byte[] code = jpeg_data.toByteArray();
                 byte[] output = Base64.encode(code, Base64.NO_WRAP);
                 String js_out = new String(output);
-                this.callbackContext.success(js_out);
+                String base64Prefix = "data:" + imageMimeType + ";base64,";
```

Comment:
why is snake_case used across this file when Java is camelCase? Could you at least make the new variables camelCase (if you want, feel free to change the others, too)

Reply:
This comment should probably be in the line below, 1274 - because the line you commented on actually is _not_ snake case.

Reply:
Oh yeah 🙈  - I've meant 1274

Reply:
I will create a new separate PR to refactor variable names properly. I totally agree with you.